examples/loaders/loader-glb.html

```diff
@@ -35,25 +35,30 @@
 
         app.scene.ambientLight = new pc.Color(0.2, 0.2, 0.2);
 
-        // Load a model file and create a Entity with a model component
+        // Load a glb file as a container
         var url = "../assets/models/playcanvas-cube.glb";
         app.assets.loadFromUrl(url, "container", function (err, asset) {
             app.start();
 
-            var entity = new pc.Entity();
-            entity.addComponent("model", {
-                type: "asset",
-                asset: asset.resource.model,
+            // create instance using model component
+            var entity1 = asset.resource.instantiateModelEntity({
```

Comment:
Again, I wouldn't confuse things. I would proceed with the render component at the recommended way here.